src/SDKs/_metadata/compute_resource-manager.txt

```diff
@@ -1,11 +1,11 @@
-2018-03-14 20:54:24 UTC
+2018-05-21 22:12:33 UTC
 
 1) azure-rest-api-specs repository information
 GitHub user: Azure
 Branch:      master
-Commit:      58409bca13e129632c26c9d3767f22f922af62dc
+Commit:      37e2f7bf51c9a8ab6079d1cfa0f0b31b475315ff
 
 2) AutoRest information
 Requested version: latest
-Bootstrapper version:    C:\Users\hylee\AppData\Roaming\npm `-- autorest@2.0.4245  
+Bootstrapper version:    C:\Users\hylee\AppData\Roaming\npm `-- autorest@2.0.4262  
```

Comment:
Please regenerate the code using `generate.ps1` script [here](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/generate.ps1)